lib/index.js

```diff
@@ -1,7 +1,28 @@
 'use strict';
 
-module.exports = {
-    jQueryBehavior: require('./behaviors/jquery-behavior.js'),
-    StickitBehavior: require('./behaviors/stickit-behavior.js'),
-    bootstrapValidatorBehavior: require('./behaviors/bootstrapvalidator-behavior.js')
-};
+(function(root, factory) {
+    if (typeof define === 'function' && define.amd) {
+        define([
+            '/bower_components/marionette.behaviors/lib/behaviors/jquery-behavior.js',
```

Comment:
Something must be wrong with our configs then. I will try on a blank RequireJS project. The relative path append to the current url and goes like `http://localhost/apps/hashhash/behaviors/jquery-behavior.js` :confused: